docs/08-logging.md

```diff
@@ -0,0 +1,65 @@
+# Log Aggregation in OpenShift
+Installation of the log aggregation solution is [documented here](https://docs.openshift.com/container-platform/4.0/logging/efk-logging-deploy.html).
+
+## Creating the Logging Project
+You can create the required Project with the following command:
+
+```sh
+oc create -f https://raw.githubusercontent.com/openshift/training/master/assets/logging-project.yaml
+```
+
+While the documentation refers to setting a `NodeSelector` at the namespace
+level, there is a [small
+bug](https://bugzilla.redhat.com/show_bug.cgi?id=1683819) that would prevent
+that from working with the infrastructure node `node-role` label. Instead, we
+will set `NodeSelectors` individually on each component.
+
+## Logging Operators
+Follow the documentation for installing the required operators.
+
+oc new-project logging
+click "show community"
+click "do not show again"
+click "show community"
+click "cluster logging" tile
+click "install"
+click "show community"
+
```

Comment:
@thoraxe The user needs to install the Elasticsearch Operator manually in addition to the Cluster Logging Operator. I point this out in the docs. Do you want to mention it here also? 
And, the list here isn't formatted correctly and the list is coming out as a single line. We would use s `.` before each step for numbers.

Reply:
This was a bad rebase. Fixing momentarily.